tests/sklearn/test_sklearn_autolog.py

```diff
@@ -422,9 +433,9 @@ def test_get_params_returns_dict_whose_key_or_value_exceeds_length_limit(long_pa
 
     X, y = get_iris()
 
-    with mock.patch("sklearn.cluster.KMeans.get_params", return_value=long_params), mock.patch(
-        "mlflow.utils._logger.warning"
-    ) as mock_warning, mlflow.start_run() as run:
+    with disable_parameter_validation(), mock.patch(
```

Comment:
nit: what about adding `disable_parameter_validation` at the test function level ?, i.e.:
```
@disable_parameter_validation
def test_get_params_returns_dict_that_has_more_keys_than_max_params_tags_per_batch():
    ...
```

Reply:
I would do this if `mock.patch("sklearn.cluster.KMeans.get_params", return_value=large_params)` was applied as a decorator.

Reply:
oh, withdraw this.`disable_parameter_validation` is not decorator.

Reply:
mock.patch can be used as decorator

Reply:
Yep, but `nullcontext` can't, right?

Reply:
yes, nullcontext can't. let's keep current code